docs/upgrade.md

```diff
@@ -107,404 +102,195 @@ nodes, and is only safe with important precautions. Therefore, it is [highly rec
-
 
 ## RabbitMQ Version Upgradability {#rabbitmq-version-upgradability}
 
-All versions starting with `3.7.27` support [rolling upgrades](#rolling-upgrades) to compatible
-later versions using [feature flags](./feature-flags).
+You can only upgrade to RabbitMQ 4.0 from RabbitMQ 3.13.x.
 
-A [full cluster stop](#full-stop-upgrades) may be required for feature version upgrades
-but such cases are rare in modern release series thanks to feature flags.
+Don't forget to [enable all stable feature flags](./feature-flags#how-to-enable-feature-flags) while still on 3.13,
+**before** attempting an upgrade to RabbitMQ 4.0, or the upgrade will fail.
 
-:::important
-As a rule of thumb, upgrade to the latest patch release in the target series,
-and then [enable all stable feature flags](./feature-flags#how-to-enable-feature-flags)
-after all cluster nodes have been upgraded.
-:::
+If you are not on RabbitMQ 3.13 yet, refer to the table below to understand your upgrade path.
 
-:::important
-When an upgrade jumps multiple release series (e.g. goes from `3.9.x` to `3.13.x`), it may be necessary to perform
-one or more intermediate upgrades first.
-
-For each intermediary upgrade, upgrade to the latest patch release in the target series,
-and then [enable all stable feature flags](./feature-flags#how-to-enable-feature-flags)
-after all cluster nodes have been upgraded.
-:::
-
-When an upgrade jumps multiple release series (e.g. goes from `3.9.x` to `3.13.x`), it may be necessary to perform
-one or more intermediate upgrades first. For each intermediary upgrade, upgrade to the latest patch release in the target series,
-and then [enable all stable feature flags](./feature-flags#how-to-enable-feature-flags)
-after all cluster nodes have been upgraded.
-
-For example, when upgrading from `3.9.x` to `3.13.x`, it would be necessary to
-
-1. First upgrade to the latest 3.10.x patch release
-2. Then to the latest patch release of 3.11.x
-3. Then to the latest patch release of 3.12.x
-4. Finally, upgrade to the latest release in the 3.13.x
-
-Don't forget to [enable all stable feature flags](./feature-flags#how-to-enable-feature-flags) every step of
-the process or the follow step may fail because some feature flags have [graduated](./feature-flags#graduation) (became mandatory).
-
-Or consider a [The Blue/Green deployment](./blue-green-upgrade) upgrade in one go.
-
-### Release Series Upgradeability with Rolling Upgrades
+<details>
+  <summary>Release Series Upgradeability</summary>
 
-Current release series upgrade compatibility with **rolling** upgrade:
+The following shows the supported upgrade paths.
 
 | From     | To     | Notes                                                         |
 |----------|--------|---------------------------------------------------------------|
+| 3.13.x   | 4.0.x  | Some feature flags **must** be enabled **before** the upgrade |
```

Comment:
Why not say "All stable feature flags must be enable before (…)"? As opposed to "some" because it gives the impression that the user is responsible for figuring out which one are to be enabled and if they make a mistake, their cluster will crash and two kittens 100 km from there will die.